src/Http/Handler/ResponseHandlerError.php

```diff
@@ -3,24 +3,38 @@
 namespace bunq\Http\Handler;
 
 use bunq\Exception\ApiException;
+use bunq\Exception\BunqException;
 use bunq\Exception\ExceptionFactory;
 use Psr\Http\Message\ResponseInterface;
 
 /**
  */
 class ResponseHandlerError extends ResponseHandlerBase
 {
+    /**
+     * Error constants.
+     */
+    const ERROR_COULD_NOT_DETERMINE_RESPONSE_ID_HEADER =
+        'The response header "X-Bunq-Client-Response-Id" or "x-bunq-client-response-id" could not be found.';
+
     /**
      * Field constants.
      */
     const FIELD_ERROR = 'Error';
     const FIELD_ERROR_DESCRIPTION = 'error_description';
+    const FIELD_BUNQ_CLIENT_RESPONSE_ID_UPPER_CASED = 'X-Bunq-Client-Response-Id';
+    const FIELD_BUNQ_CLIENT_RESPONSE_ID_LOWER_CASED = 'x-bunq-client-response-id';
 
     /**
      * Http status code constants.
      */
     const STATUS_CODE_OK = 200;
 
+    /**
+     * The index of the first item in an array.
+     */
+    const INDEX_FIRST = 0;
+
     /**
      * @param ResponseInterface $response
      *
```

Comment:
`@return`

Reply:
Invalid, as you can see this particular line has not been changed in this pr and therefore only a snippet of the doc block is show. If you look at the original file you see that this doc block is correct 🙃 see: https://github.com/bunq/sdk_php/blob/1a5188bcf53e836912ae01a82c585b89219a075f/src/Http/Handler/ResponseHandlerError.php#L41

Reply:
Ah yes, missed that one. Ignore then. 😄